src/index.ts

```diff
@@ -29,13 +30,21 @@ const run = async () => {
       ? repositoryInput.split("/")
       : [context.repo.owner, context.repo.repo];
 
+    // GITHUB_API_URL is part of GitHub Actions' built-in environment variables.
+    // See https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables.
+    const githubUrlInput = getInput("github_url");
+    const baseUrl = githubUrlInput
+      ? new URL(githubUrlInput)
+      : new URL(env.GITHUB_API_URL);
```

Comment:
I believe this is unnecessary if you have a default set to `${{ github.api_url }}`.

Reply:
Same deal, mostly duplicating the existing pattern in the code instead of introducing a new one. I'm also open to changing it around if the maintainer would prefer.